api/FileSystem.json

```diff
@@ -8,7 +8,9 @@
           "chrome": {
             "version_added": "7"
           },
-          "chrome_android": "mirror",
+          "chrome_android": {
+            "version_added": false
```

Comment:
https://mdn-bcd-collector.appspot.com/tests/api/FileSystem says no support on desktop either, so I don't trust this. I'll assume all of the FileSystem* changes that follow are similarly suspect and skip over them in review.